view/frontend/web/js/view/payment/method-renderer/two_payment.js

```diff
@@ -318,13 +346,41 @@ define([
                     quantity: item['qty'],
                     unit_price: parseFloat(item['price']).toFixed(2),
                     tax_amount: parseFloat(item['tax_amount']).toFixed(2),
-                    tax_rate: parseFloat(item['tax_percent']).toFixed(6),
+                    tax_rate: (parseFloat(item['tax_percent']) / 100 ).toFixed(6),
                     tax_class_name: '',
                     quantity_unit: config.orderIntentConfig.weightUnit,
                     image_url: item['thumbnail'],
                     type: item['is_virtual'] === '0' ? 'PHYSICAL' : 'DIGITAL'
                 });
             });
+
+
+            const orderIntentRequestBody = {
+                gross_amount: parseFloat(totals['grand_total']).toFixed(2),
+                invoice_type: config.orderIntentConfig.invoiceType,
+                currency: totals['base_currency_code'],
+                line_items: lineItems,
+                tax_subtotals : calculateTaxSubtotals(lineItems),
+                buyer: {
+                    company: {
+                        organization_number: this.companyId(),
+                        country_prefix: billingAddress.countryId,
+                        company_name: this.companyName(),
+                        website: window.BASE_URL
+                    },
+                    representative: {
+                        email: this.getEmail(),
+                        first_name: billingAddress.firstname,
+                        last_name: billingAddress.lastname,
+                        phone_number: this.getTelephone()
+                    }
+                },
+                merchant_short_name: config.orderIntentConfig.merchantShortName
+            }
+
+            console.log({lineItems : orderIntentRequestBody.line_items})
+            console.log({taxSubTotals : orderIntentRequestBody.tax_subtotals})
```

Comment:
we can probably log both in a single call. and perhaps use console.debug

Reply:
e.g. console.log({lineItems : orderIntentRequestBody.line_items, taxSubTotals : orderIntentRequestBody.tax_subtotals})

Reply:
updated